src/Jackett.Common/Indexers/FileListApi.cs

```diff
@@ -0,0 +1,216 @@
+        }
+
+        protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
+        {
+            var releases = new List<ReleaseInfo>();
+            var response = await CallProviderAsync(query);
+            releases.AddRange(TransformProviderRequest(query, response));
+
+            return releases;
+        }
+
+        private async Task<string> CallProviderAsync(TorznabQuery query)
+        {
+            var searchUrl = ApiUrl;
+            var searchString = query.GetQueryString();
+            var imdbID = query.ImdbID;
+            var cat = MapTorznabCapsToTrackers(query).FirstIfSingleOrDefault("");
+
+            var queryCollection = new NameValueCollection();
+            
+            if (imdbID != null)
+            {
+                queryCollection.Add("type", "imdb");
+                queryCollection.Add("query", imdbID);
+                queryCollection.Add("action", "search-torrents");
+            }
+            else if (!string.IsNullOrWhiteSpace(searchString))
+            {
+                queryCollection.Add("type", "name");
+                queryCollection.Add("query", searchString);
+                queryCollection.Add("action", "search-torrents");
+            } 
+            else 
+            {
+                queryCollection.Add("action", "latest-torrents");
+            }
+
+            queryCollection.Add("category", cat);
+
+
+            searchUrl += "?" + queryCollection.GetQueryString();
+
+            var response = await RequestStringWithCookiesAndRetry(searchUrl, null, null, GetAuthHeader());
+
+            return response.Content;
+        }
+
+        private List<ReleaseInfo> TransformProviderRequest(TorznabQuery query, string results)
+        {
+            var releases = new List<ReleaseInfo>();
+            JArray json = null;
+
+            try
+            {
+                json = JArray.Parse(results);
+                foreach (JObject r in json)
+                {
+                    var link = new Uri((string)r["download_link"]);
+                    var seeders = (int)r["seeders"];
+                    var publishDate = DateTimeUtil.FromFuzzyTime((string)r["upload_date"]);
+                    var release = new ReleaseInfo
+                    {
+                        Title = (string)r["name"],
+                        Comments = null,
+                        Link = link,
+                        Category = MapTrackerCatDescToNewznab((string)r["category"]),
+                        Size = (long)r["size"],
+                        Files = (long)r["files"],
+                        Grabs = (long)r["times_completed"],
+                        Seeders = seeders,
+                        PublishDate = publishDate,
+                        UploadVolumeFactor = 1,
+                        DownloadVolumeFactor = GetDownloadFactor(r),
+                        Guid = link,
+                        Peers = seeders + (int)r["leechers"]
```

Comment:
move peers close to seeds and add these:
                MinimumRatio = 1,
                MinimumSeedTime = 172800, // 48 hours